vapi/simulator/simulator.go

```diff
@@ -1898,6 +1938,17 @@ func (s *handler) deleteVM(ref *types.ManagedObjectReference) {
 	})
 }
 
+func (s *handler) reconfigVM(ref types.ManagedObjectReference, config types.VirtualMachineConfigSpec) error {
```

Comment:
nit: `reconfigureVM` better?

Reply:
I think it's ok - unexported and the API method is actually called `ReconfigVM_Task`.. not sure why chose to spell it out in the wrapper method :shrug:

Reply:
I think it's ok - unexported and the API method is actually called `ReconfigVM_Task`.. not sure why chose to spell it out in the wrapper method :shrug: